Controllers/HomeController.cs

```diff
@@ -18,22 +20,30 @@ public HomeController(ILogger<HomeController> logger)
             _logger = logger;
         }
 
-        public IActionResult Index()
+        public async Task<ActionResult> Index()
         {
-            //TODO: change this to show proper data
-            return View();
+            //All API Data
+            var CoronaApi = await CoronaApi<ApiRootObject>();
+            return View(CoronaApi);         
         }
 
-        public IActionResult Privacy()
+        // GET CovidAPI
+        [HttpGet]
+        static async Task<T> CoronaApi<T>()
         {
-            //TODO: change this to show proper data
-            return View();
+            string url = "https://api.covid19api.com/summary";
+            var client = new HttpClient();
+            var result = await client.GetStringAsync(url);
+            var DeserializeObject = JsonConvert.DeserializeObject<T>(result);
```

Comment:
put this in its own service class. Makes it testabble and makes the code less tightly coupled.

Example
https://github.com/StephenGibbsDev/MTGCore/blob/master/src/MTGCore/Services/MTGService.cs

you will need to use dependency injection.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-3.1